utils/utils.go

```diff
@@ -164,6 +174,27 @@ func Filter[T any](items []T, test func(item T) bool) []T {
 	return slices.Clip(filteredItems)
 }
 
+// MapToSlice is the inversion of SliceToMap. Order is not guaranteed as map retrieval order is not.
+func MapToSlice[T comparable, V any](m map[T]V) []T {
+	l := make([]T, len(m))
+	var i int
+	for uid := range m {
+		l[i] = uid
+		i++
+	}
+	return l
+}
+
+// SliceToMap simple utility to convert from a slice to a map generally to use for contains-like operations
```

Comment:
```suggestion
// SliceToMap converts from a slice to a map generally to use for contains-like operations
```

(It's best to avoid words like "simple" as they're [subjective](https://jameshfisher.com/2017/02/22/dont-use-simply/))